openms-wiki/reporting-bugs-and-issues.md

```diff
@@ -0,0 +1,15 @@
+Reporting Bugs and Issues
+=========================
+
+A list of known issues in the current OpenMS release can be found [here](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/nightly/html/known_dev_bugs.html). Please check if your OpenMS version matches the current version and if the bug has already been reported.
+
+In order to report a new bug, please use either our [GitHub issues system](write-and-label-github-issues.md) or [contact us](contact-us.md).
```

Comment:
> In order to report a new bug, please use either our [GitHub issues system](write-and-label-github-issues.md) or [contact us](contact-us.md).

In order to report a new bug, please create a [GitHub issue] following our bug reporting guidelines, you can also contact us.